users/reallyinvincible/Task_3/index.html

```diff
@@ -0,0 +1,229 @@
+        <b>Abraham Lincoln never does what in any photographs?</b>
+    </div>
+    <div class="options" style="background: #FFB74D;">
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions6" id="inlineRadio61"
+                   value="0">
+            <label class="form-check-label" form="inlineRadio1">Run</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions6" id="inlineRadio62"
+                   value="0">
+            <label class="form-check-label" form="inlineRadio2">Ride</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions6" id="inlineRadio63"
+                   value="0">
+            <label class="form-check-label" form="inlineRadio3">Stand</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions6" id="inlineRadio64"
+                   value="1">
+            <label class="form-check-label" form="inlineRadio3">Smile</label>
+        </div>
+    </div>
+</div>
+
+<div class="card">
+    <div class="card-header" style="background: #f44336; text-align: center; color: aliceblue;">
+        <b>What peoples name means "eaters of raw flesh"?</b>
+    </div>
+    <div class="options" style="background: #e57373;">
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions7" id="inlineRadio71"
+                   value="0">
+            <label class="form-check-label" form="inlineRadio1">Cannibal</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions7" id="inlineRadio72"
+                   value="0">
+            <label class="form-check-label" form="inlineRadio2">Masai</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions7" id="inlineRadio73"
+                   value="1">
+            <label class="form-check-label" form="inlineRadio3">Eskimo</label>
+        </div>
+        <div class="form-check form-check-inline">
+            <input class="form-check-input" type="radio" name="inlineRadioOptions7" id="inlineRadio74"
+                   value="0">
+            <label class="form-check-label" form="inlineRadio3">Red Neck</label>
+        </div>
+    </div>
+</div>
+
+<div class="jumbotron jumbotron-fluid">
+    <div class="container">
+        <h1 class="display-4" id = "score">Press Submit to know your Score</h1>
+        <p class="lead" id="summary"></p>
+    </div>
+</div>
+
+<div id="submit">
```

Comment:
I don't think you need to add a seperate `div` here. Open to suggestions here

Reply:
It is residue of my futile attempts to center the submit button. It in indeed not necessary to have a div here.